query/src/main/scala/filodb/query/exec/InProcessPlanDispatcher.scala

```diff
@@ -19,6 +19,10 @@ import filodb.core.store._
 import filodb.query.{QueryResponse, QueryResult, StreamQueryResponse}
 import filodb.query.Query.qLogger
 
+object InProcessPlanDispatcher {
```

Comment:
Michael, wondering if it is possible to have a single counter for the same ? ( we can add another tag with dispatcher="actor-plan" or dispatcher="in-process"